website/templates/util/render_node.mako

```diff
@@ -119,8 +119,8 @@
 % else:
     <li
         node_reference="${summary['id']}:${'node' if summary['primary'] else 'pointer'}"
-        class="project list-group-item list-group-item-node unavailable">
```

Comment:
Hi @caneruguz , I added a style that depends on `unavailable`  class -- https://github.com/caneruguz/osf.io/blob/stylefix/deferred-cards-3/website/static/css/style.css#L430 . I used the grey background to show that private component is not clickable.